pkg/openservicebroker/cmd/server/start.go

```diff
@@ -152,3 +185,15 @@ func (o TemplateServiceBrokerServerOptions) RunTemplateServiceBrokerServer(stopC
 	}
 	return server.GenericAPIServer.PrepareRun().Run(stopCh)
 }
+
+// these are used to set up for reading the config
+var (
+	groupFactoryRegistry = make(announced.APIGroupFactoryRegistry)
+	registry             = registered.NewOrDie("")
+	configScheme         = runtime.NewScheme()
+	configCodecs         = serializer.NewCodecFactory(configScheme)
+)
+
+func init() {
+	configinstall.Install(groupFactoryRegistry, registry, configScheme)
```

Comment:
do we actually have to install/register the config resource?  We don't anticipate anyone actually "creating"(or retrieving) a TSBConfig resource object, do we?